doc/design/OrionCKANSink.md

```diff
@@ -237,25 +253,46 @@ A configuration example could be:
     cygnusagent.sinks.ckan-sink.api_key = myapikey
     cygnusagent.sinks.ckan-sink.attr_persistence = column
     cygnusagent.sinks.ckan-sink.orion_url = http://localhost:1026
+    cygnusagent.sinks.ckan-sink.batch_size = 100
+    cygnusagent.sinks.ckan-sink.batch_timeout = 30
 
 [Top](#top)
 
-##<a name="section3"></a>Use cases
+###<a name="section2.2"></a>Use cases
 Use `OrionCKANSink` if you are looking for a database storage not growing so much in the mid-long term.
 
 [Top](#top)
 
-##<a name="section4"></a>Implementation details
-###<a name="section4.1"></a>`OrionCKANSink` class
+###<a name="section2.3"></a>Important notes
+####<a name="section2.3.1"></a>About the persistence mode
+Please observe not always the same number of attributes is notified; this depends on the subscription made to the NGSI-like sender. This is not a problem for the `row` persistence mode, since fixed 8-fields rows are upserted for each notified attribute. Nevertheless, the `column` mode may be affected by several rows of different lengths (in term of fields). Thus, the `column` mode is only recommended if your subscription is designed for always sending the same attributes, event if they were not updated since the last notification.
+
+In addition, when running in `column` mode, due to the number of notified attributes (and therefore the number of fields to be written within the Datastore) is unknown by Cygnus, the Datastore can not be automatically created, and must be provisioned previously to the Cygnus execution. That's not the case of the `row` mode since the number of fields to be written is always constant, independently of the number of notified attributes.
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/commit/7d6c2c61abac91e24c4abbe0b3abb559bf10e765